tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "declaration": true,
+    "removeComments": true,
+    // "emitDecoratorMetadata": true,
+    // "experimentalDecorators": true,
+    "lib": ["es2017"],
+    "target": "es2017",
+    "sourceMap": true,
+    "importHelpers": true,
+    "outDir": "./dist",
```

Comment:
it should be `"outDir": ".",` when importing, this (package) is not getting recognized because `package.json` main is `index.js`, and you are setting everything inside the `dist` folder.

Reply:
or, on package.json change main to `dist/index.js` .. it is able to import it this way.